src/NGettext/Catalog.cs

```diff
@@ -182,12 +182,14 @@ private string _FindTranslationFile(CultureInfo cultureInfo, string domain, stri
 
 		private string _GetFileName(string localeDir, string domain, string locale)
 		{
-			var relativePath =

```

Comment:
Oh, that makes sense.. I was thrown off when the error said that `Path.Combine(string, string, string, string)` is inaccessible due to protection level.

 I changed this because there is no `Path.DirectorySeparatorChar` in .NET Core and the next best thing should be `Path.Combine`.